packages/apps-config/src/endpoints/testing.ts

```diff
@@ -27,15 +27,15 @@ export const testChains: EndpointOption[] = [
     info: 'aleph',
     text: 'Aleph Zero Testnet',
     providers: {
-      'Aleph Zero Foundation': 'wss://ws.test.azero.dev'
+      'Aleph Zero Foundation': 'wss://ws.test.azero.dev',
+      Dwellir: 'wss://aleph-zero-testnet-rpc.dwellir.com'
```

Comment:
Have we decided to add this endpoint to our wallet? Not sure we verified it.

Reply:
Removed for now